.gitignore

```diff
@@ -1 +1,3 @@
 _cache/
+node_modules/
+screenshot-*.png
```

Comment:
Do we want to commit screenshots to git?

Reply:
I don't think so, no.

Reply:
We don't generate screenshots for the scheduled build (it probably takes about a minute to run) so they won't be committed.